Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -94,6 +97,18 @@ public BoxPlotSeries()
         public double MedianThickness { get; set; }
 
         /// <summary>
+        /// Gets or sets the size of the mean point.
+        /// </summary>
+        /// <remarks>This property is only used when ShowMeanAsDot = true.</remarks>
+        public double MeanPointSize { get; set; }
```

Comment:
So if we are doing breaking changes I would propose something like this (and the same also for mean)

```c#
public MarkerStyle MedianMarkerStyle { get; set } // default None
public LineStyle MedianLineStyle { get; set } // default Line
public double MedianLineThickness { get; set } // default 2.0
public double MedianMarkerThickness { get; set; } // default 2.0
```

But this would allow to display a Dot and a Line mixed for Mean/Median, so maybe a seperate enum (Line,  Point, Dash, Circle, Square, Star) would also be suitable.